knowledge-base/angular-report-viewer-in-multiple-tabs.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Angular Report Viewer in Multiple Tabs
+description: Sometimes when you want to display the report viewers in different tabs only in the first tab the report is rendered.
+type: how-to
+page_title: How to Display Angular Report Viewer in Multiple Tabs
+slug: angular-report-viewer-in-multiple-tabs
+position: 
+tags: 
+ticketid: 1450621
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Report Viewer</td>
+			<td>Angular</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+Sometimes when you want to display the Angular Report Viewer in multiple tabs only in the first tab the report is rendered. 
+
+## Solution
+You need to use [lazy loading](https://material.angular.io/components/tabs/overview#lazy-loading) for the tab's content:
+```
+mat-tab-group [(selectedIndex)]="selectedTab">
+             <mat-tab label="Tab Rpt Works">
+                <ng-template matTabContent>
+                  <button type="button" (click)="onLoad1Click($event)">Load</button>
+                  <tr-viewer #reportViewer
+                    [containerStyle]="viewerContainerStyle"
+                    [serviceUrl]="reportServerUrl"
+                    [viewMode]="'INTERACTIVE'"
+                    [scaleMode]="'SPECIFIC'"
+                    [scale]="1.0">
+                  </tr-viewer>
+                </ng-template>                  
+              </mat-tab>  
+ </mat-tab-group>
+```
+By default, the tab contents are eagerly loaded. Eagerly loaded tabs will initialize the child components but not inject them into the DOM until the tab is activated.
+
+## Sample Application
+The demo application could be downloaded from [here](https://github.com/telerik/reporting-samples/tree/master/AngularReportViewerInMultipleTabs). 
+Firstly open **MyRestService** project and run the [Upgrade Wizard](../ui-upgrade-wizard). Rebuild the project to recover the required
```

Comment:
it will be easier to read for users if this section uses bullets, for example
1. open **MyRestService** project 
2. Rebuild the project to recover 
3. Start the application  etc...